magda-web-client/src/config.ts

```diff
@@ -156,7 +159,7 @@ const baseExternalUrl = serverConfig.baseExternalUrl
     ? window.location.protocol + "//" + window.location.host + "/"
     : baseUrl;
 
-const fetchOptions: RequestInit =
+const credentialsFetchOptions: RequestInit =
```

Comment:
Fetch option also allows us to modify other things e.g. `headers`:
https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch
Just in case we want to globally add custom header or something. Probably should call this a more generic name?

Reply:
@t83714 yeah but I want to make it really obvious that by using these options you're making your `fetch` pass credentials! Otherwise it's too easy to think "oh a bunch of other `fetch`es use this, so my one that doesn't need credentials should too".

If we have other fetch options that do other things we should probably make them available here under other names and merge them together if a `fetch` needs more than one.